src-docs/agent.py.md

```diff
@@ -5,6 +5,10 @@
 # <kbd>module</kbd> `agent.py`
 The Jenkins agent relation observer. 
 
+**Global Variables**
+---------------
+- **AGENT_RELATION**
+- **SLAVE_RELATION**
```

Comment:
Same comment as above about non-inclusive words if possible.

Reply:
Renamed to `agent-deprecated`!